tensorflow/python/ops/numpy_ops/np_array_ops.py

```diff
@@ -1015,6 +1015,9 @@ def _split_on_axis(np_fun_name, axis):
 
   @np_utils.np_doc(np_fun_name)
   def f(ary, indices_or_sections):
+    # for 1-D array, hsplit becomes vsplit
```

Comment:
Thanks for the PR! Do you mind adding a test as well? Like when the input is a 1-D array